refractr/refractr.py

```diff
@@ -93,8 +93,7 @@ def set_headers(self, headers):
     def load_refract(self, spec):
         # this is a simple http->https redirect
         if isinstance(spec, str):
-            # nginx ingress will give a 308 status code, not 301
```

Comment:
I saw this comment as well, is this something special about the nginx-ingress controller?

Reply:
I have no idea honestly, I read the comment about 10 times trying to figure out what it means. I think it's just a comment explaining the next line.  Looking at the blame this used to be a 301 and was swapped to a 308, so I guess we're just switching back and forth between the two.

Reply:
https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#server-side-https-enforcement-through-redirect